### ISSUE TITLE
UserProfile custom attributes in lower camel case

### ISSUE THREAD

**Issue by Flo451** [okta/okta-sdk-python#294]
Hi,

Using Version 2.5.0

when querying users I get the first 200 results with the custom UserProfile attributes named as expected (matching with what the API returns).
Going to the next page of 200 results suddenly the custom attributes are lower camel case.

Could you please have a look?

**Comment by laura-rodriguez** [comment]
Hi @Flo451,

Thanks for reporting this. Can you please share the minimal piece of code to reproduce this issue? 

A Fiddler log is also desirable to help investigate the issue.

Thanks!

**Comment by Flo451** [comment]
After further testing it appears the lower camel case attribute is a duplicate. So in my case the custom attribute named `JobFamily` suddenly also has a duplicate named `jobFamily`.

```
import asyncio
from okta.client import Client as OktaClient

config = {

}
okta_client = OktaClient(config)

async def main():
    query_parameters = {'filter': 'status eq "ACTIVE"'}
    users, resp, err = await okta_client.list_users(query_parameters)
    allusers=[]
    while True:
        allusers+=users
        if resp.has_next():
            users, err = await resp.next()
        else:
            break
    # users = [user for user in allusers if hasattr(user.profile, "JobFamily")]
    for count, user in enumerate(allusers):
        print(count, end=' ')
        if hasattr(user.profile, "JobFamily"):
            print('check',end='| ')
        else:
            print('nope',end='| ')
        if hasattr(user.profile, "jobFamily"):
            print('check', end=' ')
        else:
            print('nope',end='| ')
        print()
        # print(count, hasattr(user.profile, "JobFamily"), hasattr(user.profile, "jobFamily"))


if __name__=='__main__':
    loop = asyncio.get_event_loop()
    loop.run_until_complete(main())
```

From row 200 we get a duplicated lower camel case attribute.

```
grep 200 -C10 output
190 check| nope|
191 check| nope|
192 check| nope|
193 check| nope|
194 check| nope|
195 check| nope|
196 check| nope|
197 check| nope|
198 check| nope|
199 check| nope|
200 check| check
201 nope| nope|
202 nope| nope|
203 check| check
204 check| check
205 check| check
206 check| check
207 check| check
208 check| check
209 check| check
210 check| check
```

**Comment by bretterer** [comment]
@Flo451 Thank you for that information.  Have you confirmed with raw api calls that you are not getting `jobFamily` in your responses?

**Comment by Flo451** [comment]
Yes, the custom attribute is `JobFamily`. The duplication also happens for other custom attributes. The duplicated values are not available in the raw api call results.

**Comment by bretterer** [comment]
Resolved in 2.9.0
